### PR TITLE
RDSC-1204-document-log-rotations-and-configuration-on-the-pacemaker-page

### PR DESCRIPTION
**Pull request by loriotpiroloriol**

### CODE REVIEW

File: content/rdi/installation/pacemaker.md [hunk 1]
```diff
@@ -282,6 +282,12 @@ To launch the Debezium Server container use the podman service of Pacemaker (Pod
 sudo pcs resource create dbz_server ocf:heartbeat:podman image=docker.io/debezium/server allow_pull=yes run_opts="-v $PWD/conf:/debezium/conf"
 ```
 
+If you are using the Docker resource agent, you should also specify some logging options:
```
Comment (spicy-sauce) [inline content/rdi/installation/pacemaker.md:285]: Can we add a link to the logging documentation for full reference of that options?

**Review by spicy-sauce — CHANGES_REQUESTED** [review]

File: content/rdi/installation/pacemaker.md
[the remark this answers is not in the document]
Comment (loriotpiroloriol) [reply content/rdi/installation/pacemaker.md:1]: Done

File: content/rdi/installation/pacemaker.md (hunk 1 above)
Comment (loriotpiroloriol) [reply content/rdi/installation/pacemaker.md:285]: Do you mean [this](https://github.com/RedisLabs/redislabs-docs/blob/RDSC-1204-document-log-rotations-and-configuration-on-the-pacemaker-page/content/rdi/installation/debezium-server-deployment.md#containerized-deployment)? Or the Docker documentation [here](https://docs.docker.com/config/containers/logging/configure/)?

Comment (spicy-sauce) [reply content/rdi/installation/pacemaker.md:285]: I think the Docker documentation is preferred as it's more comprehensive and covers everything.

Comment (loriotpiroloriol) [reply content/rdi/installation/pacemaker.md:285]: Done

File: content/rdi/installation/pacemaker.md
[the remark this answers is not in the document]
Comment (kaitlynmichael) [reply content/rdi/installation/pacemaker.md:1]: For Redis Enterprise documentation, we use sentence case for titles and headings (only first word and proper nouns are capitalized). Here's a helpful link to the Google style guide: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

[the remark this answers is not in the document]
Comment (spicy-sauce) [reply content/rdi/installation/pacemaker.md:1]: Thanks, we had a different convention in RDI documentation.
@loriotpiroloriol we should revert this based on this guideline, sorry for the misleading.
@yaronp68 FYI

[the remark this answers is not in the document]
Comment (loriotpiroloriol) [reply content/rdi/installation/pacemaker.md:1]: Done

**Review by elena-kolevska — APPROVED** [review]

**Pull request merged**